lostfiles.py

```diff
@@ -116,11 +131,109 @@ def parse_args() -> argparse.Namespace:
     )
     return parser.parse_args()
 
+def packages():
```

Comment:
What do you think about defining a dict with those paths, for example:
```
PKG_PATHS = {
    "app-emulation/libvirt": [
       *glob("/etc/libvirt/nwfilter/*.xml"),
       *glob("/etc/libvirt/qemu/*.xml"),
       ...
    ]
}
```

that would make `packages()` a bit more general and only contain exceptions like `check_process("systemd")`

Reply:
If you're right, I hadn't thought about that
I'm not good at python, I'm learning at full speed :-)
Then I get to it.
Thanks